app/src/main/java/de/westnordost/streetcomplete/quests/SharedFilters.kt

```diff
@@ -0,0 +1,12 @@
+package de.westnordost.streetcomplete.quests
+
+import de.westnordost.streetcomplete.osm.MAXSPEED_TYPE_KEYS
+
+val anyMaxSpeedTagKey = "~${(MAXSPEED_TYPE_KEYS + "maxspeed").joinToString("|")}"
+
+val onlyInANoneMaxSpeed30OrLessZone = """
```

Comment:
Typo: **_Non_**, not _None_.

The common name for these things is just [30 zone](https://en.wikipedia.org/wiki/30_km/h_zone) (or 20 mph zone). Since the actual speed limit is variable, depending on the sign, in StreetComplete, you'll find that in some places it is called "slow zone".

So, I'd prefer "notInSlowZone", or better, "inSlowZone" and negate it in usage.

Reply:
But it's more than that. It's also not a simple maxspeed value.